src/default-config.json

```diff
@@ -7,5 +7,5 @@
   "asn": "enter the ASNs of interest here",
   "peerIp": "enter peer IP",
   "connector": "enter connector or connectors of interests",
-  "windowSize": "360"
+  "windowSize": "2"
```

Comment:
assume this should be changed back to a higher value (but a windowSize of 2 or even 1 is a great test of boundary conditions in the windowing/anomaly detection code, even if not useful for real workloads)